### PR TITLE
launcher

### PR DESCRIPTION
**Pull request by AgustinBRodriguez**
Changes to documentation

Added a Launcher App page to the User manual 
Added a Deprecated page to the Reference section and nested Brain v1 and Filter App.
made the necessary changes to 
https://github.com/farm-ng/Amiga-support-issues/issues/231#issuecomment-2050188556

### CODE REVIEW

**Comment by AgustinBRodriguez** [comment]
@guilhermedemouraa can you please add a few lines to describe/explain robot localization and track following?. The serial interface field in the ntrip section also needs a brief description. Let me know if you have any questions. Thank you

**Review by guilhermedemouraa — CHANGES_REQUESTED** [review]

File: website/docs/apps/launcher/README.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+## Apps
+
+Please visit the following links for an in-depth
+exploration of the [Autoplot App](/docs/apps/autoplot_app),
+[Recorder App](/docs/apps/recorder_app),
+[Filemanager App](/docs/apps/file_manager_app),
+and [Camera App](/docs/apps/camera_app).
+
+## Settings
+
```
[the remark this answers is not in the document]
Comment (AgustinBRodriguez) [reply website/docs/apps/launcher/README.md:23]: Added

File: website/docs/apps/filter_app/README.md [hunk 2]
```diff
@@ -3,6 +3,11 @@ id: filter-app
 title: Filter App
 ---
 
+:::warning
+The configuration settings and robot geometry found in this page are now part
+of the launcher app and can be found under the settings tab.
```
[the remark this answers is not in the document]
Comment (AgustinBRodriguez) [reply website/docs/apps/filter_app/README.md:8]: added

File: website/docs/apps/launcher/README.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+The WiFi Manager is where you will select which network you would like to join.
+Upon selecting a given network, you will be prompted to input a password.
+After all the fields have been satisfied your connection to WiFi will be established.
+
+![Screenshot from 2024-04-12 10-20-18](https://github.com/farm-ng/amiga-dev-kit/assets/133177230/4eab6af0-fb5f-46f8-aa96-9b974acce051)
+
+### Services
+
+For an in dept exploration of the Amiga Services follow this [link](/docs/concepts/canbus_service/).
```
[the remark this answers is not in the document]
Comment (AgustinBRodriguez) [reply website/docs/apps/launcher/README.md:45]: modified

[the remark this answers is not in the document]
Comment (AgustinBRodriguez) [reply website/docs/apps/launcher/README.md:45]: will do

File: website/docs/apps/launcher/README.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+**User**: Username for accessing the base station.
+
+**Password**: Corresponding password for the above username.
+
+**Serial Interface**:
+
+![carrsoln](https://github.com/farm-ng/amiga-dev-kit/assets/133177230/c206e401-3fdf-41a8-805c-2f55a90a8c2d)
+
+:::tip
```
[the remark this answers is not in the document]
Comment (AgustinBRodriguez) [reply website/docs/apps/launcher/README.md:167]: clarified

**Pull request merged**